workspaces/global-header/plugins/global-header/package.json

```diff
@@ -57,6 +59,7 @@
     "@backstage/cli": "^0.29.0",
     "@backstage/core-app-api": "^1.15.2",
     "@backstage/dev-utils": "^1.1.4",
+    "@backstage/plugin-search-common": "^1.2.16",
```

Comment:
Nit: Not needed here because it's already a dependency above.

@ciiay can you please remove this in your next PR?

Reply:
@christoph-jerolimov  I actually added it because it failed on `yarn lint` as following, removing it will block it from being pushed.

```
Lint failed in plugins/global-header

✘  https://google.com/#q=%40backstage%2Fno-undeclared-imports

     @backstage/plugin-search-common is declared in devDependencies, but should be moved to dependencies in plugins/global-header/package.json
```